### DISCUSSION TITLE
Job apply page is broken

### DISCUSSION THREAD

**Discussion by arafatkatze** [vercel/community#4353] [Account Issue]
### Question

<img width="551" alt="image" src="https://github.com/vercel/community/assets/11155207/277389ba-b592-4fd0-a9d6-783ef3a0a876">

**Comment by amyegan** [comment]
Hi @arafatkatze. Is this error happening when you click the "Careers" link to visit https://vercel.com/careers? Or is it happening for a different reason?

**Comment by arafatkatze** [comment]
<img width="958" alt="image" src="https://github.com/vercel/community/assets/11155207/7bdfcb21-47a7-4f0e-9c9f-1890bdcaae5d">

Happens when i click apply on this posting after filling everything https://vercel.com/careers/software-engineer-ai-us-4808680004

**Reply by amyegan** [reply]
Thanks for sharing that, @arafatkatze. We weren't able to reproduce the problem. Do you see any additional info in the dev tools console or network tab? That might reveal more about what is going wrong.

Missing or invalid values are a common cause of form submission errors. Please double check that you have provided valid input for all required fields. 🙏
